### ISSUE TITLE
Missing `aria-labels` for Stock Tools buttons

### ISSUE THREAD

**Issue by jedrzejruta** [highcharts/highcharts#23011]
#### Expected behaviour
Descriptive aria-labels are visible by screen readers for each Stock Tools button.

#### Actual behaviour
All buttons are read out as "button" or "clickable button", without any descriptive information or label.

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/pu48nwg7/

#### Product version
Since introduction of Stock Tools (v7.0.0+)

![Image](https://github.com/user-attachments/assets/dfe36a06-29aa-48ef-b432-543b4d5e1271)

**Comment by jedrzejruta** [comment]
Workaround: https://jsfiddle.net/BlackLabel/91qhwf5v/
